### PR TITLE
Add secDevLabs to the list of offline applications

### PR DESCRIPTION
**Pull request by Krlier**
The goal of this PR is to add [secDevLabs](https://github.com/globocom/secDevLabs) to the list of offline applications.

### CODE REVIEW

File: src/data/offline.json [hunk 1]
```diff
@@ -733,6 +733,28 @@
...
+			"Dart/Flutter"
+		],
+		"references": [
+			{
+				"name": "guide",
+				"url": "https://github.com/globocom/secDevLabs"
+			}
+		],
+		"author": "Globo",
+		"notes": "Repository with many intentionally vulnerable web applications. Includes attack naratives and docker options for each app.",
```
Comment (kingthorin) [inline src/data/offline.json:755]:
```suggestion
		"notes": "Repository with many intentionally vulnerable web applications. Includes attack narratives and docker options for each app.",

```

Comment (Krlier) [reply src/data/offline.json]: Hi, @kingthorin! Thanks for taking the time to review this PR.

I have fixed the typo you pointed out.

**Review by kingthorin — APPROVED** [review]
Thanks

**Comment by kingthorin** [comment]
Deployed: https://github.com/OWASP/www-project-vulnerable-web-applications-directory/pull/38 should be live in a minute or two.

**Pull request merged**